lib/plugins/aws/deploy/index.js

```diff
@@ -104,6 +106,11 @@ class AwsDeploy {
       'aws:deploy:deploy:updateStack': async () => {
         if (this.serverless.service.provider.shouldNotDeploy) return;
         await this.updateStack();
+        log.notice(
+          `\n${chalk.hex('#fd5750')('✔')} Service deployed to stack ${this.serverless
```

Comment:
We have to repeat this `chalk.hex` in multiple places - what do you think of defining our custom chalk instance that already has these defined e.g. in `utils` ? It's very error prone if we'd decide to replace the colour to a bit different shade

Reply:
We definitely should do that.  Let me prepare something in `@serverless/utils`.